lib/slack-ruby-bot/support/commands_helper.rb

```diff
@@ -0,0 +1,102 @@
+require 'singleton'
+
+module SlackRubyBot
+  class CommandHelpAttrs
```

Comment:
Refactor this one in its own file. I think it could use better namespacing, like `SlackRubyBot::Commands::Help::Attrs`, but I don't feel too strongly about this.